.github/workflows/crowdin-i18n-client-ui-download.yml

```diff
@@ -90,14 +90,39 @@ jobs:
         with:
           github-token: ${{ secrets.CROWDIN_CAMPERBOT_PAT }}
           script: |
+            const branchExists = await github.repos.getBranch({
+              owner: 'freeCodeCamp',
+              repo: 'freeCodeCamp',
+              branch: 'i18n-sync-client'
+            }).catch(err => {
+              console.info("Branch does not exist. Likely no changes in download?");
+            });
+            if (!branchExists || branchExists.status !== 200) {
+              return;
+            }
+            const pullRequestExists = await github.pulls.list({
+              owner: 'freeCodeCamp',
+              repo: 'freeCodeCamp',
+              head: 'freeCodeCamp:i18n-sync-client'
+            });
+            if (pullRequestExists.data.length) {
```

Comment:
Does `github.pulls.list` always return an array so that referencing the `length` property will not be a problem if not?

Reply:
I believe so. [Here is a run](https://github.com/nhcarrigan/actions-testing/runs/2041044606?check_suite_focus=true) which found no open pull requests, and an empty array was returned.